src/cocotb_tools/runner.py

```diff
@@ -1609,6 +1609,103 @@ def _test_command(self) -> List[_Command]:
         return cmds
 
 
+class Dsim(Runner):
+    """Implementation of :class:`Runner` for DSim.
+
+    * Does not support the ``gui`` argument to :meth:`.test`.
+    * Does not support the ``pre_cmd`` argument to :meth:`.test`.
+    """
+
+    supported_gpi_interfaces = {"verilog": ["vpi"]}
+
+    def _simulator_in_path(self) -> None:
+        if shutil.which("dsim") is None:
+            raise SystemExit("ERROR: dsim executable not found!")
+
+    def _get_include_options(self, includes: Sequence[PathLike]) -> _Command:
+        return [f"+incdir+{_as_tcl_value(str(include))}" for include in includes]
```

Comment:
Just checking this is correct? You would be the one that knows. Do the include paths need to be formatted as TCL strings?

Reply:
Nop, this was a mistake.